images/linux/scripts/installers/nodejs.sh

```diff
@@ -20,3 +30,4 @@ apt-get update
 apt-get install -y --no-install-recommends yarn
 
 invoke_tests "Tools" "Node.js"
+invoke_tests "CLI.Tools" "Vercel CLI"
```

Comment:
maybe migrate the test to the Tools Node.js?

Reply:
Thank you, moved Vercel CLI test to Tools Node.js